modules/virt-vm-wizard-fields-web.adoc

```diff
@@ -51,10 +51,10 @@ endif::[]
 |Operating System
 |
 ifdef::virtualmachine[]
-|The primary operating system that is selected for the virtual machine in the template. You cannot edit this field when creating a virtual machine from a template.
+|The operating system that is selected for the virtual machine in the template. You cannot edit this field when creating a virtual machine from a template.
 endif::[]
 ifdef::vmtemplate[]
-|The primary operating system that is selected for the virtual machine. Selecting an operating system automatically selects the default *Flavor* and *Workload Type* for that operating system.
+|The operating system that is selected for the virtual machine. Selecting an operating system automatically selects the default *Flavor* and *Workload Type* for that operating system.
```

Comment:
@davidvossel do you know what "flavor" in this context means?

Reply:
The next line in the table shows what "Flavor" means:


Flavor
small, medium, large, tiny, Custom
Presets that determine the amount of CPU and memory allocated to the virtual machine. The presets displayed for Flavor are determined by the operating system.

Does that help?

Reply:
My questions about that are unrelated I guess. No reason to hold this PR up for that.